packages/venia-concept/src/actions/checkout/__tests__/asyncActions.spec.js

```diff
@@ -169,21 +169,29 @@ describe('submitAddress', () => {
             cart: {},
             directory: { countries }
         }));
-
         await expect(submitAddress(payload)(...thunkArgs)).rejects.toThrow(
             'guestCartId'
         );
     });
+    test('submitAddress thunk dispatches action on incorrect state(region code)', async () => {
+        const invalidState = 'any_text';
+        const incorrectAddressMessage = `State "${invalidState}" is not an valid state abbreviation.`;
+        const incorrectAddressPayload = { incorrectAddressMessage };
+        const submitPayload = {
+            type: 'address',
+            formValues: { region_code: invalidState }
+        };
 
-    test('submitAddress thunk throws if payload is invalid', async () => {
```

Comment:
Did you mean to delete this test case? Is it invalid?

Reply:
@supernova-at I've swapped out throwing error to dispatching action "shippingMethod.incorrectAddress", so submitAddress won't be throwing errors if address payload is invalid